sceptre/template_handlers/file.py

```diff
@@ -24,12 +24,8 @@ def schema(self):
         }
 
     def handle(self):
-        project_path = self.stack_group_config.get("project_path")
         input_path = Path(self.arguments["path"])
-        if input_path.is_absolute():
```

Comment:
I think what we might want to check would be something like
```
project_templates = (Path(self.stack_group_config.get("project_path")) / 'templates').absolute()
if input_path.absolute().exists():
    path = str(input_path)
elif (project_templates / input_path).exists():
    path = str(project_templates / input_path)
else:
    # the file doesn't exist... so we should probably blow up?
```

Reply:
@jfalkenstein A few issues with this notably that the non-existence of the file could simply mean the user has forgotten to put the file there. But I think my refactored version renders this discussion moot. Let me know your thoughts.